cg/services/delivery_message/messages/rna_analysis_scout_message.py

```diff
@@ -0,0 +1,59 @@
+from cg.services.delivery_message.messages.delivery_message import DeliveryMessage
+from cg.services.delivery_message.messages.utils import (
+    get_caesar_delivery_path,
+    get_scout_link,
+)
+from cg.store.models import Case
+
+
+class RNAAnalysisScoutMessage(DeliveryMessage):
+    def __init__(self, store):
+        super().__init__()
+        self.store = store
+
+    def create_message(self, cases: list[Case]) -> str:
```

Comment:
I added these `get_case_message` and `get_cases_message` as part of the class since they require the store (instead of passing it as an argument). Let me know if this should be changed!

Reply:
For me it is ok